tasks/coverage.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+const $ = require('gulp-load-plugins')()
+
+const utils = require('../src/utils')
+const config = require('../config/webpack')
+
+module.exports = {
+  fn (gulp, done) {
+    gulp.task('pre-test', function () {
+      return gulp.src([
+        'src/**/*.js'
+      ]).pipe($.istanbul())
+        .pipe($.istanbul.hookRequire())
+    })
+
+    gulp.task('mocha', ['pre-test'], () => {
+      return gulp.src([
+        'test/node.js',
+        'test/**/*.spec.js'
```

Comment:
We instrument the `src` on l12 with Istanbul, and run tests with mocha. We don't need to instrument the tests folder.